### PR TITLE
Update ArtQRCode.cs

### PR DESCRIPTION
**Pull request by Tragen**
Fixed dot size calculation where the size is divided instead of multiplied.

### CODE REVIEW

**Comment by Shane32** [comment]
Previously the pixel size factor does not appear to have any effect.

With pixel size factor of 0.7: (default is 0.8)

Before:

test

After:

test

Demo of default size factor of 0.8:

test

It would seem that this PR should be merged, assuming this is the intent of the pixel size factor.  However, any user code will now generate with smaller dots than previously, as previously it acted as though pixelSizeFactor was 1.  Test hashes will need to be updated.

**Comment by Shane32** [comment]
Closes:
- #463

File: QRCoderTests/ArtQRCodeRendererTests.cs [hunk 1]
```diff
@@ -96,7 +96,7 @@ public void can_render_artqrcode_from_helper()
             var bmp = ArtQRCodeHelper.GetQRCode("A", 10, Color.Black, Color.White, Color.Transparent, QRCodeGenerator.ECCLevel.L);
 
             var result = HelperFunctions.BitmapToHash(bmp);
-            result.ShouldBe("57ecaa9bdeadcdcbeac8a19d734907ff");
+            result.ShouldBe("6c1bb8257349b29f76e50c3fa8abd9c9");
```
Comment (Shane32) [inline QRCoderTests/ArtQRCodeRendererTests.cs:99]: Do you want to change the default size from 0.8 to 1.0 on the helper as well?  Then this hash would have remained the same.

Comment (Shane32) [reply QRCoderTests/ArtQRCodeRendererTests.cs:99]: See ArtQRCode.cs line 286

**Review by Shane32 — APPROVED** [review]
Looks good to me

File: QRCoderTests/ArtQRCodeRendererTests.cs (hunk 1 above)
Comment (codebude) [reply QRCoderTests/ArtQRCodeRendererTests.cs]: I've overseen it. Was just on my way fixing it, when I saw your comment.

**Pull request merged**